tests/www/test_views.py

```diff
@@ -270,6 +270,73 @@ def test_prefill_form_null_extra(self):
         cmv.prefill_form(form=mock_form, pk=1)
 
 
+class TestMultiDuplicateConnectionModelView(TestBase):
```

Comment:
Please don't create a new test class -- instead add methods to the existing TestConnectionModelView